akka-docs/src/main/paradox/additional/deploying.md

```diff
@@ -21,7 +21,29 @@ features from Akka Management for smooth rolling updates.
  
 ### Resource limits
 
-To avoid CFS scheduler limits, it is best not to use `resources.limits.cpu` limits, but use `resources.requests.cpu` configuration instead.
+To avoid CFS scheduler limits, it is best not to use `resources.limits.cpu` limits, but use `resources.requests.cpu`
+configuration instead.
+
+For memory, it's recommended to set both `resources.requests.memory` and `resources.limits.memory` to the same value.
+The `-XX:InitialRAMPercentage` and `-XX:MaxRAMPercentage` JVM options can be used to set the heap size relative to the
+memory limit.
```

Comment:
Could we also mention something about off heap and that we recommend 70% MaxRAMPercentage (possible more for larger memory limit)?

Reply:
Ok, updated to include a 70% recommendation.